content/03.categorize.md

```diff
@@ -231,6 +231,58 @@ a need for better pathologist-computer interaction techniques that will allow
 combining the power of deep learning methods with human expertise and lead to
 better-informed decisions for patient treatment and care.
 
+### Text applications in healthcare
+
+Due to the rapid growth of scholarly publications and electronic medical/health records, biomedical text mining has become increasingly important in recent years.
+The main tasks in biological and clinical text mining include, but are not limited to, named entity recognition, relation/event extraction, and information retrieval (Figure @fig:biotm).
+
+![Deep learning applications, tasks and models based on NLP perspectives.](images/biotm.png){#fig:biotm width="100%"}
+
+Machine learning is widely used in many of aforementioned tasks where engineering the optimal set of features is a must.
+However it is often difficult to know what features should be extracted, even with domain knowledge.
+Deep learning addresses this challenge by enabling "computers to build complex
+concepts out of simpler ones [@tag:goodfellow2016deep]".
```

Comment:
Question for @dhimmel on manubot : is it ok to have this here and
> A recent book from Goodfellow et al. [@url:http://www.deeplearningbook.org/] covers these in detail.
in the `02.intro.md` file?

Reply:
Manubot can handle this and will report it during the build:
https://travis-ci.org/greenelab/deep-review/builds/302503795#L753

We'll be okay as long as the URLs match.  For maintainability though, I think it is preferable to use a tag everywhere once we introduce it, i.e., change the intro to use the Goodfellow tag as well before merging this.  That way if we ever find a DOI or something for the tag we'll stay consistent.